pages/index.jsx

```diff
@@ -13,9 +15,9 @@ const Home = () => {
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <h1 className="text-center text-9xl underline font-semibold text-white">
-        Unit A
-      </h1>
+      <Wrapper>
+        <Projects/>
+      </Wrapper>
```

Comment:
Don't call the wrapper in this file, call it instead inside the component in which it's being used